octobot/src/server/github_handler.rs

```diff
@@ -322,6 +367,7 @@ impl Handler for GithubHandler {
             pr_merge,
             repo_version,
             force_push,
+            team_members_cache: TeamsCache::new(Duration::new(3600, 0)),
```

Comment:
imo, the named constructors are a bit easier to read.

```suggestion
            team_members_cache: TeamsCache::new(Duration::from_secs(3600)),
```